package.json

```diff
@@ -70,6 +70,7 @@
   ],
   "dependencies": {
     "amd-name-resolver": "0.0.5",
+    "babel-plugin-compact-reexports": "^0.1.0",
     "broccoli-babel-transpiler": "^5.6.1",
```

Comment:
We need to remove our direct usage of `broccoli-babel-transpiler` and instead swap to the new API in ember-cli-babel@6 that is documented [here](https://github.com/babel/ember-cli-babel#addon-usage). This also allows us to remove `DEFAULT_BABEL_OPTIONS`.